prinz/src/main/scala/pl/touk/nussknacker/prinz/util/exceptions/Assertion.scala

```diff
@@ -0,0 +1,8 @@
+package pl.touk.nussknacker.prinz.util.exceptions
+
+object Assertion {
+
+  @inline def assertIllegal(check: Boolean, message: String): Unit = if (!check) {
+    throw new IllegalArgumentException(message)
+  }
+}
```

Comment:
Do we need it? There's only one call to `assertIllegal`.

Reply:
for now only one, let's use it if you want. It seems to be a good function (and place to create other assertions functions) for me that can make code more concise